### ISSUE TITLE
Remove offline.html Implementations

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#56]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
PWA is to make as many pages available offline as possible and NOT redirect people to an offline page. Hence, we can safely remove it.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
The offline.html feature is removed from the package.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
The half-baked offline.html feature is still made available from the package.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 210c73d7f2d015605c8b0bc1c2662a2e9c7698e9

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
